tests/test_vpc.rb

```diff
@@ -21,33 +21,27 @@
 # TestVpc - Minitest for Vpc node utility class
 class TestVpc < CiscoTestCase
   def setup
+    @skip_unless_supported = 'vpc'
```

Comment:
`@skip_unless_supported` is a class instance variable, so this is incorrect - this line belongs at the class level, not inside the `setup` method:

```ruby
class TestVpc < CiscoTestCase
  @skip_unless_supported = 'vpc'

  def setup
    super
    no_feature_vpc
  end

  def teardown
    no_feature_vpc
    super
  end
```